Practice/Shiny_App.R

```diff
@@ -0,0 +1,80 @@
+# Load packages-----------------------------------------------------------------
+library(tidyverse)
+library(shiny)
+library(shinythemes)
+library(shinyWidgets)
+
+
+# Build UI----------------------------------------------------------------------
+ui <- fluidPage(
+  
+  titlePanel("Mental Illness & Socioeconomic Status"),
+  
+  fluidRow(
+    column(5, offset = 1,
+           sliderTextInput("income", "Household Income",
```

Comment:
This is where naming the arguments for functions come handy. In the below, the `inputId = income` so we will pass this into the `server <- ..` part of the script as `filter(`Household Income` == input$income)`
```suggestion
           sliderTextInput(inputId = "income", label = "Household Income",
```